ansible/roles/vtgate/defaults/main.yml

```diff
@@ -20,5 +20,5 @@ vtgate_grpc_port: '15306'
 
 enable_vtgate: True
 
-vtgate_query_cache_size: 0
+vtgate_query_cache_size: 1000
```

Comment:
Interested to know why we are are changing the cache size
- Is 1000 the default for vtgate ?

Reply:
The default is 5000 as shown in the code below:
https://github.com/vitessio/vitess/blob/25baa46125279255ef70bb2d78a2c3738e29f5e7/go/cache/cache.go#L76-L80

When running the Ansible roles with the inventory files we normally use (`./ansible/koz-*.yml`), we set `vttablet_query_cache_size` to a non-zero value as highlighted in this example: https://github.com/vitessio/arewefastyet/blob/4f6626497f46cf7128ec06578444caf714af7c68/ansible/koz-inventory-unsharded-test.yml#L43

However, when we run Molecule, we do not take into account these inventory files. Thus, vtgate (and vttablet) role uses their own `vttablet_query_cache_size` default value (defined in `./ansible/roles/*/defaults/*.yml).

Having a value equal to 0 causes the server to crash as defined in the below code:
https://github.com/vitessio/vitess/blob/25baa46125279255ef70bb2d78a2c3738e29f5e7/go/cache/ristretto/cache.go#L164-L165

Reply:
Both vttablet and vtgate are using `vt*_query_cache_size `.